src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,147 @@
+package duke;
+
+import duke.exceptions.*;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+
+import java.util.Scanner;
+
+import static duke.exceptions.UserInputException.inputExceptionType;
+import static duke.exceptions.UserInputException.inputExceptionType.INVALID_TASK_TYPE;
+
+public class Duke {
+    //String constants
+    private static final String LOGO = " ____        _        \n" + "|  _ \\ _   _| | _____ \n" + "| | | | | | | |/ / _ \\\n" + "| |_| | |_| |   <  __/\n" + "|____/ \\__,_|_|\\_\\___|\n";
```

Comment:
This is definitely longer than the maximum length you can have in one line. Consider separating it with `+` and using more lines on it. If you are unsure on the indentation needed for it, take a look at the java coding standards.